internal/webhookcert/webhook_cert_test.go

```diff
@@ -28,10 +28,16 @@ var (
 		Name:      "telemetry-webhook-cert",
 		Namespace: systemNamespace,
 	}
-	name        = "telemetry-validating-webhook.kyma-project.io"
-	webhookName = types.NamespacedName{
-		Name: name,
+	nameValidatingWebhook = "telemetry-validating-webhook.kyma-project.io"
+	validatingWebhookName = types.NamespacedName{
```

Comment:
`nameValidatingWebhook` and `validatingWebhookName` are very confusing.
Maybe we rename them to `validatingWebhookName` and `validatingWebhookNamespacedName`?

Reply:
renamed